src/test/java/io/xpring/xrpl/WalletTest.java

```diff
@@ -197,4 +201,19 @@ public void testVerifyEmptyStringBadSignature() throws XrpException {
 
     assertFalse(wallet.verify(message, signature));
   }
+
+  @Test
+  public void testRandomWalletFromFaucet() throws Exception {
+    // GIVEN a new, randomly generated wallet that is funded by the Testnet faucet
+    Wallet wallet = XrpTestUtils.randomWalletFromFaucet();
+
+    // WHEN the wallet is examined
+    // THEN it exists
+    assertNotNull(wallet);
+
+    // AND it has a non-zero balance
+    XrpClient xrpClient = new XrpClient("test.xrp.xpring.io:50051", XrplNetwork.TEST);
+    BigInteger balance = xrpClient.getBalance(wallet.getAddress());
+    assertTrue(balance.compareTo(new BigInteger("0")) == 1);
```

Comment:
I think it might be slightly more idiomatic to use `signum` here: https://www.geeksforgeeks.org/biginteger-signum-method-in-java/